includes/uploader.js

```diff
@@ -121,17 +135,64 @@ function check_file_content(isValidFile) {
 
         reader.readAsText(file);
     }
+
 }
 
 
 
-// Check file content 
+function check_columns_names(isValidContent) {
+    if (isValidContent) {
+        // Define the names accepted for the columns
+        const columns_names_reference_list = [
+            "Statement Type",
+            "Attribute",
+            "Deontic",
+            "Aim",
+            "Direct Object",
+            "Type of Direct Object",
+            "Indirect Object",
+            "Type of Indirect Object",
+            "Activation Condition",
+            "Execution constraint",
+            "Or else\r"
+
+        ];
+
+        // Array to store unrecognized column names
+        const unrecognizedColumns = [];
+
+        // Iterate through columnNames and check correspondence
+        columnNames.forEach(function (name, index) {
+            console.log(name);
+
+            console.log(columnNames);
+            if (!columns_names_reference_list.includes(name)) {
+                // Add the unrecognized column name to the list
+                unrecognizedColumns.push(name);
+            }

```

Comment:
If we're OK with users submitting CSVs with valid column names, but in different orders, then we can make this check easier by using a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set):

```javascript
const expectedColumnNames = new Set([
    "Statement Type",
    ...
])

var unrecognizedColumns = Set(columnNames).difference(expectedColumnNames)
```